<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigDialog</class>
 <widget class="QDialog" name="ConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>476</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configuration de Oook (Open Orang-outan Knowledge)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="sourceTitle">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Source</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="calibreLabel">
       <property name="text">
        <string>Bibliothèque Calibre</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="calibreLayout">
       <item>
        <widget class="QLineEdit" name="calibre"/>
       </item>
       <item>
        <widget class="QPushButton" name="calibreButton">
         <property name="text">
          <string>…</string>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="databaseLabel">
       <property name="text">
        <string>Base de données Calibre</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <layout class="QHBoxLayout" name="databaseLayout">
       <item>
        <widget class="QLineEdit" name="database"/>
       </item>
       <item>
        <widget class="QPushButton" name="databaseButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="destinationTitle">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Destination</string>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="catalogLabel">
       <property name="text">
        <string>Catalogue</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="catalogLayout">
       <item>
        <widget class="QLineEdit" name="catalog"/>
       </item>
       <item>
        <widget class="QPushButton" name="catalogButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="templatesTitle">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Affichage</string>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Titre</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="title">
       <property name="text">
        <string>Oook (Open Orang-outan Knowledge)</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="templatesLabel">
       <property name="text">
        <string>Modèles</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <layout class="QHBoxLayout" name="templatesLayout">
       <item>
        <widget class="QLineEdit" name="templates"/>
       </item>
       <item>
        <widget class="QPushButton" name="templatesButton">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="itemsPerPageLabel">
       <property name="text">
        <string>Éléments par page</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QSpinBox" name="itemsPerPage">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="specialValueText">
        <string/>
       </property>
       <property name="suffix">
        <string/>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="thumbnailLabel">
       <property name="text">
        <string>Taille des miniatures</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <layout class="QHBoxLayout" name="thumbnailLayout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <item>
        <widget class="QSpinBox" name="thumbnailL">
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="thumbnailSeparator">
         <property name="text">
          <string>×</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="thumbnailH">
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>144</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="coverLabel">
       <property name="text">
        <string>Taille des couvertures</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <layout class="QHBoxLayout" name="coverLayout">
       <property name="spacing">
        <number>-1</number>
       </property>
       <item>
        <widget class="QSpinBox" name="coverL">
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>380</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="coverSeparator">
         <property name="text">
          <string>×</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="coverH">
         <property name="suffix">
          <string> px</string>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
         <property name="value">
          <number>542</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="copyTitle">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Copie</string>
       </property>
       <property name="margin">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <layout class="QHBoxLayout" name="copyLayout" stretch="0,0,0,0,0">
       <item>
        <widget class="QRadioButton" name="copyTrivial">
         <property name="text">
          <string>Superficielle</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="copySeparatorLeft">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="copyNormal">
         <property name="text">
          <string>Normale</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="autoExclusive">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="copySeparatorRight">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QRadioButton" name="copyFull">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Complète</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>calibre</tabstop>
  <tabstop>calibreButton</tabstop>
  <tabstop>database</tabstop>
  <tabstop>databaseButton</tabstop>
  <tabstop>catalog</tabstop>
  <tabstop>catalogButton</tabstop>
  <tabstop>itemsPerPage</tabstop>
  <tabstop>thumbnailL</tabstop>
  <tabstop>thumbnailH</tabstop>
  <tabstop>coverL</tabstop>
  <tabstop>coverH</tabstop>
  <tabstop>copyTrivial</tabstop>
  <tabstop>copyNormal</tabstop>
  <tabstop>copyFull</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>256</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>324</x>
     <y>442</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>calibreButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>selectCalibre()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>444</x>
     <y>8</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>databaseButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>selectDatabase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>76</y>
    </hint>
    <hint type="destinationlabel">
     <x>512</x>
     <y>7</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>catalogButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>selectCatalog()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>518</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>417</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>templatesButton</sender>
   <signal>clicked()</signal>
   <receiver>ConfigDialog</receiver>
   <slot>selectTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>507</x>
     <y>216</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectCalibre()</slot>
  <slot>selectDatabase()</slot>
  <slot>selectCatalog()</slot>
  <slot>selectTemplate()</slot>
 </slots>
</ui>
